src/styles/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
```

Comment:
Make sure that the last element of the row don't have margin-right
![image](https://github.com/mate-academy/layout_product-cards/assets/96409563/3dd4738d-a3cc-4c18-8909-0d0fd160f36e)

Reply:
done, thanks)